### PR TITLE
Implement content

### PR DESCRIPTION
**Pull request by charmychancey**
Fill the pages (main page, about me, see me, reach me) with content. Implement css to make pages more mobile friendly/resizable.

### CODE REVIEW

File: portfolio/src/main/webapp/see-me.html [hunk 1]
```diff
@@ -27,5 +27,56 @@ <h1>My Portfolio</h1>
...
+        </div>
+        <div class="entry">
+            <img src="images/food.jpg" alt="food.jpg">
+            <p>I love to cook!...whether it's getting my hands on delicious
+                ingredients or trying new, promising recipes.</p>
+        </div>
+        <div class="entry">
+            <img src="images/tkd.jpg" alt="tkd.jpg">
+            <p>I started practicing taekwondo at UC Davis. I
+                really appreciate the power behind it's flashy kicks!</p>
```
Comment (snazzorama) [inline portfolio/src/main/webapp/see-me.html:64]: it's -> its

File: portfolio/src/main/webapp/about-me.html [hunk 2]
```diff
@@ -26,6 +26,48 @@ <h1>My Portfolio</h1>
...
+                    <img src="images/about-me-img1.jpeg" alt="profile-pic.jpeg" class="circular-portrait">
+                </div>
+                <div class="about-me-content">
+                    <div class="three-rows">
+                        <div class="row-element">
+                            <h2>This is me!</h2>
+                            <br>
+                            Hello! My name is Chance Lau, and I study computer science at the
+                            University of California - Davis.
+                            <br>A list of 10 things about me:</br>
```
Comment (snazzorama) [inline portfolio/src/main/webapp/about-me.html:42]: I think `<br>` should just be used to force a line break. Where you have actual paragraphs, it's better to use `<p>`...`</p>`.

You could also consider using `<ol>` for your numbered list below.

Comment (robotsaurus) [reply portfolio/src/main/webapp/about-me.html:42]: +1 to ol

File: portfolio/src/main/webapp/see-me.html (hunk 1 above)
Comment (charmychancey) [reply portfolio/src/main/webapp/see-me.html]: Thanks!

File: portfolio/src/main/webapp/about-me.html (hunk 2 above)
Comment (charmychancey) [reply portfolio/src/main/webapp/about-me.html]: Done!

**Pull request merged**